hack/update-codegen.sh

```diff
@@ -86,6 +103,7 @@ gen-client() {
     --input-base "" \
     --input k8s.io/test-infra/prow/apis/prowjobs/v1 \
     --output-package k8s.io/test-infra/prow/client/clientset
+  copyfiles "./prow/client/clientset" "*.go"
```

Comment:
🤔 these might need to be '*.go' (single quotes instead of double), although it appears to be working as-is... 🤔

Reply:
Glob expansion won't happen inside of either quotation mark so you're fine as is